dataprofiler/profilers/profile_builder.py

```diff
@@ -1842,21 +1842,23 @@ def _get_correlation(self, clean_samples, batch_properties):
         :type batch_properties: dict()
         """
         columns = self.options.correlation.columns
+        column_ids = list(range(len(self._profile)))
+        if columns is not None:
+            column_ids = [self._col_name_to_idx[col_name][0] for col_name in columns]
```

Comment:
what happens if a pd.DF has two columns with the same name? Is there a way to specify one of the two columns? Thoughts?

Reply:
Good callout, I don't think this current implementation handles that. Should be an easy feature to add I think.

Reply:
we either 
- handle processing dataframes with two or more column names that are the same 
OR 
- we raise an error that all column names in the dataframe must be unique

Reply:
**what happens if a pd.DF has two columns with the same name?**
If there are columns with the same name, the current implementation only takes the first of the repeating columns. Could implement the functionality to use all of the repeating columns.

**Is there a way to specify one of the two columns?**
In the current implementation no. Would need to work on CorrelationOptions to add that functionality.